modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/Andes.java

```diff
@@ -525,27 +522,28 @@ public void moveMessageToDeadLetterChannel(DeliverableAndesMetadata message, Str
         MessagingEngine.getInstance().moveMessageToDeadLetterChannel(message, destinationQueueName);
     }
 
-    /**
-     * Get content chunk from store.
-     *
-     * @param messageId   id of the message
-     * @param offsetValue chunk id
-     * @return message content
-     * @throws AndesException
-     */
-    public AndesMessagePart getContent(long messageId, int offsetValue) throws AndesException {
-        return MessagingEngine.getInstance().getContent(messageId, offsetValue);
-    }
+//    /**
+//     * Get content chunk from store.
+//     *
+//     * @param messageId   id of the message
+//     * @param offsetValue chunk id
+//     * @return message content
+//     * @throws AndesException
+//     */
+//    public AndesMessagePart getContent(long messageId, int offsetValue) throws AndesException {
+//        return MessagingEngine.getInstance().getContent(messageId, offsetValue);
+//    }
 
     /**
      * Get content chunks for a list of message ids from store.
      *
      * @param messageIdList list of messageIds
+     * @param destinationQueueName
      * @return map of message id:content chunk list
      * @throws AndesException
      */
-    public LongObjectHashMap<List<AndesMessagePart>> getContent(LongArrayList messageIdList) throws AndesException {
-        return MessagingEngine.getInstance().getContent(messageIdList);
+    public HashMap<Long ,List<AndesMessagePart>> getContent(LongArrayList messageIdList, String destinationQueueName) throws AndesException {
+        return MessagingEngine.getInstance().getContent(destinationQueueName , messageIdList);
```

Comment:
I think storageQueueName is a better name. Topics does not have destination queue name.